src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,34 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationService.class);
+        ShoppingCartService shoppingCartService = (ShoppingCartService) injector
+                .getInstance(ShoppingCartService.class);
+
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                    fastAndFurious.getId(), LocalDate.now()));
+
+        try {
+            User bob = authenticationService.register("bob@gmail.com", "qwerty");
+            System.out.println(bob);
+            System.out.println(shoppingCartService.getByUser(bob));
+            shoppingCartService.addSession(movieSessionService
+                    .get(tomorrowMovieSession.getId()), bob);
+            ShoppingCart shoppingCart = shoppingCartService.getByUser(bob);
+            System.out.println(shoppingCart);
+            OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+            System.out.println(orderService.completeOrder(shoppingCart));
+            System.out.println(shoppingCart);
+            System.out.println(orderService.getOrdersHistory(bob));
```

Comment:
Verify that the `getOrdersHistory` method in `OrderService` retrieves the correct data. The checklist suggests using inner joins or separate queries to fetch order and ticket information, avoiding `FetchType.EAGER` due to performance concerns .